pydicom/values.py

```diff
@@ -281,7 +481,24 @@ def convert_TM_string(byte_string, is_little_endian, struct_format=None):
 
 
 def convert_UI(byte_string, is_little_endian, struct_format=None):
-    """Read and return a UI values or values"""
+    """Return a decoded 'UI' value.
+
+    Elements with VR of 'UI' may have a non-significant padding null ``0x00``.
```

Comment:
null -> zero?

Reply:
The standard refers to it as a trailing null

http://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_6.2.html